registration/src/correspondence_rejection_one_to_one.cpp

```diff
@@ -53,7 +53,7 @@ pcl::registration::CorrespondenceRejectorOneToOne::getRemainingCorrespondences(
             pcl::registration::sortCorrespondencesByMatchIndexAndDistance());
 
   remaining_correspondences.resize(input.size());
-  int index_last = -1;
+  pcl::index_t index_last = -1;
```

Comment:
static cast here since we're converting -1 (signed) to potentially unsigned

Reply:
Changed to `UNAVAILABLE` from pcl_base.h